CHANGELOG.md

```diff
@@ -15,7 +15,10 @@
 - Added a [guide](docs/devctr-image.md) for updating the dev container image.
 - Added a new API call, `PUT /mmds/config`, for configuring the
   `MMDS` with a custom valid link-local IPv4 address.
-
+- Added JSON response format support for MMDS guest applications requests.
+- Added MMDS `403 BadRequest` failure response when trying to issue a `PATCH`
```

Comment:
This should be in another section: either `# Changed` or `# Fixed` because you are changing a behavior, not adding a new one.

nit: I would rephrase it a bit to something like: `PATCH` request on `/mmds` before the data store is initialized return `403 BadRequest`.